frontend/src/packages/sections/sections-api.ts

```diff
@@ -14,14 +19,14 @@ type Constructor = {
 
 class SectionsApi extends HttpApi {
   public constructor({ baseUrl, http, storage }: Constructor) {
-    super({ path: ApiPath.SITES, baseUrl, http, storage });
+    super({ path: '', baseUrl, http, storage });
```

Comment:
move it back

Reply:
And how am I supposed to refer to different ApiPaths (Sites and Sections in this case). Or should I move getSectionsBySiteId method to SitesApi?

Reply:
right
sections api - for sections
sites api - for sites